### ISSUE TITLE
Hyperparameter Search

### ISSUE THREAD

**Issue by nomomon** [nomomon/text-2-emoji-kitchen#10]
Use a grid search to find the best combination of hyper-parameters (PCA dim red size, learning rate, optimizer, activation function, number of nodes in the hidden layer, etc.)

**Comment by nomomon** [comment]
Not necessary for the MVP (just make sure the mode is actually learning, better than random)
